src/components/create-game/create-game.component.html

```diff
@@ -1,5 +1,8 @@
 <mat-toolbar color="primary">
-    <span>Spiel erstellen</span>
+  <button mat-icon-button (click)="session.logout()" #tooltip="matTooltip" matTooltip="Logout" matToolTipPosition="below" class="back-icon">
```

Comment:
I would put logout on the right

Reply:
I agree, but dont know when i have time.
Look at the app.component.desktop.html, i implented right aligned buttons there if you want to complete it.